OpenKeychain/src/main/java/org/sufficientlysecure/keychain/securitytoken/SecurityTokenConnection.java

```diff
@@ -948,12 +950,29 @@ public SecurityTokenInfo getTokenInfo() throws IOException {
         String url = getUrl();
         byte[] pwInfo = getPwStatusBytes();
 
-        return SecurityTokenInfo.create(fingerprints, aid, userId, url, pwInfo[4], pwInfo[6]);
+        TransportType transportType = mTransport.getTransportType();
```

Comment:
@Valodim I found a very strange problem here only happening for Fidesmo cards: In the current order of commands everything works, but if I first get the transport type including ``isFidesmoToken()`` and only afterwards get the fingerprints ``rawFingerprints`` will be null. Do you have an idea what could cause this?